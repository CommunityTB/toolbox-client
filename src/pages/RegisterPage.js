import React, { Component } from 'react';

class RegisterPage extends Component {

    state = {
    }

    render() {
        return (
          <p>PUT REGISTRATION FORM HERE</p>
        );
    }
}

export default RegisterPage;
