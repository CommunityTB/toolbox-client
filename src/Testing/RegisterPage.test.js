import React from 'react'
import ReactDOM from 'react-dom';
import { BrowserRouter } from 'react-router-dom'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import RegisterPage from '../pages/RegisterPage';

describe(`RegisterPage component`, () => {
   it('renders without crashing', () => {
     const div = document.createElement('div');
     ReactDOM.render(<BrowserRouter><RegisterPage /></BrowserRouter>, div);
     ReactDOM.unmountComponentAtNode(div);
   });
 
   it('renders a RegisterPage by default', () => {
     const wrapper = shallow(<RegisterPage />)
     expect(toJson(wrapper)).toMatchSnapshot()
   })
 })