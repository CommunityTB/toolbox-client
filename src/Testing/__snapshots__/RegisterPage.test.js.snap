// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RegisterPage component renders a RegisterPage by default 1`] = `
<section
  className="Register-page"
>
  <RegisterForm
    onCreateNewUser={[Function]}
  />
</section>
`;
